src/main/java/edu/cornell/kfs/sys/CUKFSParameterKeyConstants.java

```diff
@@ -20,6 +20,8 @@
     public static final String PDP_CUSTOMER_MISSING_ADVICE_RETURN_EMAIL = "PDP_CUSTOMER_MISSING_ADVICE_RETURN_EMAIL";
 
     public static final String CONFIDENTIAL_ATTACHMENT_FILENAME_PATTERNS = "CONFIDENTIAL_ATTACHMENT_FILENAME_PATTERNS";
+    
+    public static final String OLTP_LOCK_OVERRIDE_LIST = "OLTP_LOCK_OVERRIDE_LIST";

```

Comment:
It would be good for this constant to reflect the fact that it is a list of net ids/user names. Maybe OLTP_LOCK_OVERRIDE_USERNAME_LIST?